### PR TITLE
ECOM-3403 UI tweaks ahoy

### PR DESCRIPTION
**Pull request by AlasdairSwan**
WIP

### CODE REVIEW

File: programs/static/templates/program_details.underscore [hunk 1]
```diff
@@ -5,7 +5,7 @@
                 <span class="js-model-value copy copy-large emphasized"><%- name %></span>
                 <label for="program-name" class="sr-only"><%= gettext('Name') %></label>
                 <input type="text" value="<%- name %>" id="program-name" class="program-name field-input is-hidden" name="name" data-field="name" maxlength="64" required>
-                <button class="js-enable-edit btn btn-edit">
+                <button class="js-enable-edit btn btn-edit" data-tooltip="Edit program title">
```
Comment (jimabramson) [inline programs/static/templates/program_details.underscore]: should tooltips have `gettext` also ?

Comment (AlasdairSwan) [inline programs/static/templates/program_details.underscore]: Yes they should, thanks for catching that.

**Comment by jimabramson** [comment]
have not reviewed visually but apart from one nit code LGTM

**Comment by AlasdairSwan** [comment]
Cool, @marcotuts if you have time this afternoon you could review the visual updates at my desk?

**Comment by AlasdairSwan** [comment]
@marcotuts gave his :+1:

**Pull request merged**